### ISSUE TITLE
None of the Neighborhood Study Rezoning Commitments have been deduplicated

### ISSUE THREAD

**Issue by AmandaDoyle** [NYCPlanning/db-knownprojects#174]
`SELECT * FROM reviewed_clusters."fix" WHERE source = 'Neighborhood Study Rezoning Commitments';` shows the correct project-ids for these records; however, these project ids are not the same in outputted KPDB.  What happened?

**Comment by mgraber** [comment]
Error came from hot-fix applied in #111.

See the two attempts at logic for hot-fix in the comments. First attempt had a different join field for this N Study Rezoning Commitments, where as the final logic we used to fix the problem did not. N Study Rezoning Commitment fixes should have been merged into combined.fix using record_name, not record_id. Between Mark's 2020 cluster review and the time of the hot-fix, the N Study Rezoning Commitment record_ids changed because of another fix: #101.

To fix this problem, update combined.fix and combined with cluster_ids and other review fields from Mark's review:

```sql
UPDATE reviewed_clusters.fix a
SET cluster_id = b.cluster_id,
	sub_cluster_id = b.sub_cluster_id,
	review_initials = b.review_initials,
	review_notes = b.review_notes
FROM reviewed_clusters."2020" b
WHERE (a.record_name = b.record_name
AND a.source = 'Neighborhood Study Rezoning Commitments'
AND b.source = 'Neighborhood Study Rezoning Commitments')

UPDATE combined a
SET cluster_id = b.cluster_id,
	sub_cluster_id = b.sub_cluster_id,
        project_id = b.cluster_id || '-' || b.sub_cluster_id,
	review_initials = b.review_initials,
	review_notes = b.review_notes
FROM reviewed_clusters."2020" b
WHERE (a.record_name = b.record_name
AND a.source = 'Neighborhood Study Rezoning Commitments'
AND b.source = 'Neighborhood Study Rezoning Commitments')
```
This impacts the the automatic unit deduplication happening in step 04.

**Comment by mgraber** [comment]
Fixed using the above query. Currently, the fixed version is in kpdb.test.
